trio_websocket/__init__.py

```diff
@@ -498,21 +500,38 @@ def port(self):
         return self._port
 
     async def listen(self, *, task_status=trio.TASK_STATUS_IGNORED):
-        ''' Listen for incoming connections. '''
+        '''
+        Listen for incoming connections.
+
+        The listener and all connection tasks are spawned in an internal
+        nursery. This coroutine runs until cancelled.
```

Comment:
I believe trio lingo is "task".  In trio style, coroutines are considered a detail of Python async and not passed around.  Await is use immediately on any coroutine.

Reply:
It's true we avoid the word "coroutine" as being jargony and ambiguous. Here I'd just say "function", or "async function" if i wanted to emphasize that its call signature has an `await`. I'd use "task" if I wanted to emphasize it running as a separate "thread" of execution – so it's not a task until someone calls it via `nursery.start(_soon)`.